UI/window-basic-main.cpp

```diff
@@ -3765,8 +3765,6 @@ void OBSBasic::CheckForUpdates(bool manualUpdate)
 	updateCheckThread.reset(new AutoUpdateThread(manualUpdate));
 	updateCheckThread->start();
 #endif
-
-	UNUSED_PARAMETER(manualUpdate);
```

Comment:
`manualUpdate` is used in a `#ifdef`

```
#ifndef _WIN32
UNUSED_PARAMETER(manualUpdate)
#endif
```

Reply:
Fixed

Reply:
The separate `#ifndef` was to cover Linux and macOS at the same time.